git-delete-branch: add page (#6011)
tldr-pages/tldr: pages/common/git-graft.md

# git graft

> > Merge commits from a specific branch into another and delete the source branch.

Reviewer: ```suggestion
> Merge commits from a branch into another and delete the source branch.
```

> Part of `git-extras`
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-graft>.

> - Merge all commits not present on destination branch from source branch, and delete source branch:

Reviewer: ```suggestion
- Merge all commits not present on the target branch from the source branch, and delete the source branch:
```

> `git graft {{source_branch}} {{destination_branch}}

Reviewer: ```suggestion
`git graft {{source_branch}} {{target_branch}}
```
